### ISSUE TITLE
Azure DevOps credentials stopped working in 1.89.1.0

### ISSUE THREAD

**Issue by tibitoth** [ForkIssues/TrackerWin#2015]
For an Azure DevOps remote GitFork always ask for password for every command. This worked before the update.

![image](https://github.com/fork-dev/TrackerWin/assets/8333960/486c23f2-652e-41bc-81df-53d20ec44ac2)

```
fatal: Cannot determine the organization name for this 'dev.azure.com' remote URL. Ensure the `credential.useHttpPath` configuration value is set, or set the organization name as the user in the remote URL '{org}@dev.azure.com'.
error: unable to read askpass response from 'C:\Users\toth.tibor\AppData\Local\Fork\app-1.89.1\Fork.AskPass.exe'
bash: line 1: /dev/tty: No such device or address
error: failed to execute prompt script (exit code 1)
fatal: could not read Password for 'https://autsoft@dev.azure.com': No such file or directory
```

**Comment by DanPristupov** [comment]
> Ensure the `credential.useHttpPath` configuration value is set

Try to open terminal and Fork and run:

```
git config --global credential.useHttpPath true
```

Then fetch again.

**Comment by tibitoth** [comment]
Thanks! It's a good workaround but I don't know how broke this

**Comment by DanPristupov** [comment]
I think there's something wrong with the latest git. I'm investigating that.

Let's keep this open in a case other people also have this issue.

**Comment by CommanderAlchemy** [comment]
This just occurred on multiple computers for us
1.89.1.0
I myself worked around it by using ssh instead of https

**Comment by bdemasie** [comment]
Happening on multiple computers for my team too. Can anyone post a workaround for dummies?

**Comment by bdemasie** [comment]
Workaround for dummies:

1. From your menu, go to _Repository > Open in Console_.
2. Type this exactly:     git config --global credential.useHttpPath true
3. Press enter
4. Push/pull again

**Comment by RichardC64** [comment]
This work for me. Strange

**Comment by MartinZikmund** [comment]
Having the same problem trying to push to GitHub repos

**Comment by Dan-Korn** [comment]
Thanks, the "git config --global credential.useHttpPath true" workaround worked for me as well.

**Comment by DanPristupov** [comment]
We just released a `1.89.2` hotfix which sets `credential.useHttpPath` automatically.
